### ISSUE TITLE
Login Facebook 

### ISSUE THREAD

**Issue by piseysen** [abacritt/angularx-social-login#780]
now can not login with Facebook, thank.

**Comment by maxnag** [comment]
Not true. This  application works correctly. I can login via Google, FB
Maybe problem in your FB developer account.
Need more info

**Comment by piseysen** [comment]
@maxnag , thanks. now it work

**Comment by petkoo55** [comment]
@piseysen what was your issue? After upgrade to Angular 18, Facebook is not working, it just show new pop up window and it disappears... But I realized that when autologin is enabled, it also displays popup and then when I refresh page (F5) it login me
